arch/arm/core/cortex_m/fault.c

```diff
@@ -195,7 +195,7 @@ static bool memory_fault_recoverable(z_arch_esf_t *esf)
 }
 
 #if defined(CONFIG_ARMV6_M_ARMV8_M_BASELINE)
-/* HardFault is used for all fault conditions on ARMv6-M. */
+/* hard_fault is used for all fault conditions on ARMv6-M. */
```

Comment:
Change this to HardFault , because we (probably) refer to the HardFault exception in the ARM terminology

Reply:
i.e. not the function.